src/constants.ts

```diff
@@ -81,6 +81,7 @@ export const RequestStatus = {
 export enum VerificationStatus {
   FAILED = 'Fail - Unable to verify',
   SUCCESS = 'Pass - Verified',
+  PARTIAL = 'Pass - Verified (with a metadata mismatch)',
```

Comment:
I like it but I would make it more clear adding "Partially".

`PARTIAL = 'Pass - Partially verified (with a metadata mismatch)',`

Reply:
👍